aospy/calc.py

```diff
@@ -173,6 +174,14 @@ def __init__(self, proj=None, model=None, run=None, ens_mem=None, var=None,
             dtype_in_vert=self.dtype_in_vert,
             dtype_in_time=self.dtype_in_time, intvl_out=self.intvl_out)
 
+        if not var.def_time:
+            for reduction in dtype_out_time:
+                if reduction in _TIME_DEFINED_REDUCTIONS:
+                    msg = ("Var {0} has no time dimension "
+                           "for the given time reduction "
+                           "{1}".format(var.name, reduction))
+                    logging.info(msg)
```

Comment:
Per https://github.com/spencerahill/aospy/issues/202#issuecomment-348496875, let's raise an exception here instead of just logging a message (probably a `ValueError` is appropriate here).

Reply:
for testing, are the only possible values for `dtype_out_time` are `None`, some subset of `['av', 'std, 'ts', 'reg.av', 'reg.std', 'reg.ts']`, or individual strings of: `'av', 'std, 'ts', 'reg.av', 'reg.std', 'reg.ts'`?

Reply:
Yes 👍